cmd/validator/validator.go

```diff
@@ -307,6 +331,29 @@ func setFlagFromEnvIfNotSet(flagName string, envVar string) error {
 	return nil
 }
 
+// Build exclude-file-type list from file-type values
+func getExcludeFileTypesFromFileTypes(fileTypesPtr *string) string {
+	validTypes := make([]string, 0, len(filetype.FileTypes))
+
+	for _, t := range filetype.FileTypes {
+		validTypes = append(validTypes, t.Name)
+	}
+
+	validExcludeTypes := misc.ArrToMap(validTypes...)
+
+	for _, t := range strings.Split(*fileTypesPtr, ",") {
+		delete(validExcludeTypes, t)
+	}
+
+	excludeFileTypes := make([]string, 0, len(validExcludeTypes))
+
+	for fileType := range validExcludeTypes {
+		excludeFileTypes = append(excludeFileTypes, fileType)
+	}
```

Comment:
This looks complicated

Isn't something that could be used in Go standard "slice" package

Reply:
I couldn't anything useful in slices packages that can be used here. Would adding comments make it more readable? @ccoVeille

Reply:
Agreed - I think this can be simplified a bit

Reply:
Refactored it to use `slices.DeleteFunc`.

Reply:
Thanks, as you can see the code is now way clearer